src/providers/simulation-provider.ts

```diff
@@ -20,6 +20,8 @@ export enum SimulationStatus {
   Unattempted = 0,
   Failed = 1,
   Succeeded = 2,
+  InsufficientBalance = 3,
+  NotApproved = 4,
```

Comment:
When is unattempted used now? Should we remove?

Reply:
Removed it and replaced with NotSupported (for cases like Celo)